<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Maens-MacBook-Pro.local" failures="1" tests="1" name="runner.TestNGRunner" time="13.560" errors="0" timestamp="2021-11-03T12:05:12 AST" skipped="0">
  <testcase classname="runner.TestNGRunner" name="runScenario" time="13.560">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at stepDefinition.TestFeatures.search_for(TestFeatures.java:59)
at âœ½.I search for test(src/test/resources/testFeatures/Test Feature 1.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runner.TestNGRunner -->
